src/Compilers/CSharp/Test/Syntax/Parsing/ParsingTests.cs

```diff
@@ -145,10 +150,20 @@ internal void UsingExpression(string text, params DiagnosticDescription[] expect
         /// Parses given string and initializes a depth-first preorder enumerator.
         /// </summary>
         protected SyntaxTree UsingTree(string text, CSharpParseOptions? options = null)
+        {
+            return UsingTree(text, options, verifyErrors: false, expectedErrors: null);
```

Comment:
it would be great if we could file a design-debt item to eliminate usages of 'verifyErrors: false', or at least move any such usages to clearly-and-ugly-named methods, e.g. `UsingTreeIgnoreDiagnostics`.

Reply:
[https://github.com/dotnet/roslyn/issues/61586](https://github.com/dotnet/roslyn/issues/61586)